node/node_test.go

```diff
@@ -35,7 +35,7 @@ func TestNewNode(t *testing.T) {
 	}
 	decider := quorum.NewDecider(quorum.SuperMajorityVote)
 	consensus, err := consensus.New(
-		host, values.BeaconChainShardID, leader, blsKey, decider,
+		host, shard.BeaconChainID, leader, blsKey, decider,
```

Comment:
Actually this field should be "shard id", so if you want to refactor, better be "BeaconShardID". Because ChainID is a different thing than ShardID.

Reply:
right, confusing, done